flash/image/embedding/model.py

```diff
@@ -77,9 +77,9 @@ class ImageEmbedder(AdapterTask):
 
     def __init__(
         self,
-        training_strategy: str,
-        head: str,
-        pretraining_transform: str,
+        training_strategy: Optional[str] = "default",
```

Comment:
No need for the optional typing as I guess we don't support `None` here:
```suggestion
        training_strategy: str = "default",
```